easybuild/easyconfigs/t/torchvision/torchvision-0.16.0-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,34 @@
+name = 'torchvision'
+version = '0.16.0'
```

Comment:
I think we want to use the PyTorch-bundle approach instead. I combined all the current torch* PRs into https://github.com/easybuilders/easybuild-easyconfigs/pull/19987

Reply:
@lexming any thoughts on this?